src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -321,6 +321,11 @@ public enum EvalMetricType
         [TGUI(Label = "Ranking Label Gain")]
         public string CustomGains = "0,3,7,15,31,63,127,255,511,1023,2047,4095";
 
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Parameter for the sigmoid function. Used only in binary and multiclassova " +
```

Comment:
>multiclassova [](start = 115, length = 13)

This is a nice description for direct instantiation of lightgbm, in our case it need to be translated to multiclass trainer with useSoftMax = true

Reply:
Can you clarify why?

Reply:
this is only place where we expose this "multiclassova" for user, and it's unclear what it's means. if you want user to read documentation on lightgbm github, you can just put link to internal lightgbm documentation here, but I think it would be somewhat rude.

---
In reply to: [208724218](https://github.com/dotnet/machinelearning/pull/667#discussion_r208724218) [](ancestors = 208724218)

Reply:
It means multi-class ova. Seems pretty clear to me.

Reply:
Ok, I'm doing something like
pipeline.Add(new LightGbmClassifier(...));
Is it multiclassova or not? 
Should I set sigmoid to something other than default value or it's useless?

Or should I do pipeline.Add(new Ova(LightGbmBinaryClassifier)) instead?
Is this multiclassova?

---
In reply to: [208727571](https://github.com/dotnet/machinelearning/pull/667#discussion_r208727571) [](ancestors = 208727571)